docs/glossary.rst

```diff
@@ -0,0 +1,297 @@
+        Given a :math:`n\times m` cost matrix :math:`C`, denotes the set of
+        pairs of vectors
+
+        .. math::
+
+          D(C):= \{f \in\mathbb{R}^n, g \in \mathbb{R}^m
+          | f_i + g_j \leq C_{ij}\}.
+
+    dualize
+        Within the context of optimization, the process of converting a
+        constrained optimization problem into an unconstrained one, by
+        transforming constraints into penalty terms in the objective function.
+
+    entropy-regularized optimal transport
+        The data of the entropy regularized OT (EOT) problem is parameterized by
+        a cost matrix :math:`C` of size ``[n, m]`` and two vectors :math:`a,b`
+        of non-negative weights of respective size ``n`` and ``m``.
+        The parameters of the EOT problem consist of three numbers
+        :math:`\varepsilon, \tau_a, \tau_b`.
+
+        The optimization variables are a pair of vectors of sizes ``n`` and
+        ``m`` denoted as :math:`f` and :math:`g`.
+
+        Using the reparameterization for :math:`\rho_a` and
+        :math:`\rho_b` using
+        :math:`\tau_a=\rho_a /(\varepsilon + \rho_a)` and
+        :math:`\tau_b=\rho_b /(\varepsilon + \rho_b)`, the EOT optimization
+        problem reads:
+
+        .. math::
+
+          \max_{f, g} - \langle a, \phi_a^{*}(-f) \rangle -  \langle b,
+          \phi_b^{*}(-g) \rangle - \varepsilon \left(\langle e^{f/\varepsilon},
+          e^{-C/\varepsilon} e^{g/\varepsilon} \rangle -|a||b|\right)
+
+        where :math:`\phi_a(z) = \rho_a z(\log z - 1)` is a scaled entropy, and
+        :math:`\phi_a^{*}(z) = \rho_a e^{z/\varepsilon}`, its Legendre transform
+        :cite:`sejourne:19`.
+
+        That problem can also be written, instead, using positive scaling
+        vectors `u`, `v` of size ``n``, ``m``, and the kernel matrix
+        :math:`K := e^{-C/\varepsilon}`, as
+
+        .. math::
+
+          \max_{u, v >0} - \langle a,\phi_a^{*}(-\varepsilon\log u) \rangle
+          + \langle b, \phi_b^{*}(-\varepsilon\log v) \rangle -
+          \langle u, K v \rangle
+
+        Both of these problems can be written with a *primal* formulation, that
+        solves the :term:`unbalanced` optimal transport problem with a variable
+        matrix :math:`P` of size ``n`` x ``m`` and positive entries:
+
+        .. math::
+
+          \min_{P>0} \langle P,C \rangle +\varepsilon \text{KL}(P | ab^T)
+          + \rho_a \text{KL}(P\mathbf{1}_m | a)
+          + \rho_b \text{KL}(P^T \mathbf{1}_n | b)
+
+        where :math:`\text{KL}` is the generalized Kullback-Leibler divergence.
+
+        The very same primal problem can also be written using a kernel
+        :math:`K` instead of a cost :math:`C` as well:
+
+        .. math::
+
+          \min_{P>0}\, \varepsilon \text{KL}(P|K)
+          + \rho_a \text{KL}(P\mathbf{1}_m | a)
+          + \rho_b \text{KL}(P^T \mathbf{1}_n | b)
+
+        The *original* OT problem taught in linear programming courses is
```

Comment:
I think it would be great to have a separate OT entry, the EOT is long.

Reply:
Just noticed you have "optimal transport" term here, would then consider moving it there.

Reply:
> I think it would be great to have a separate OT entry, the EOT is long.

It's already there, as 

```This results in a standard min cost flow problem also called the :term:`Kantorovich problem`.``` which closes the paragraph